utils/test/clearing_operations_tests/ClearingOperationsPointTargetTestCase.py

```diff
@@ -0,0 +1,122 @@
+
+class ClearingOperationsPointTargetTestCase(unittest.TestCase):
+
+    toolboxUnderTest = None # Set to Pro or ArcMap toolbox at runtime
+
+    pointTarget = None
+    inputArea = None
+    output = None
+
+
+    scratchGDB = None
+    def setUp(self):
+        if Configuration.DEBUG == True: print("         ClearingOperationsPointTargetTestCase.setUp")
+
+        ''' Initialization needed if running Test Case standalone '''
+        Configuration.GetLogger()
+        Configuration.GetPlatform()
+        ''' End standalone initialization '''
+        self.toolboxUnderTest = Configuration.clearingOperationsToolboxPath + \
+            Configuration.GetToolboxSuffix()
+
+        UnitTestUtilities.checkArcPy()
+        DataDownload.runDataDownload(Configuration.clearingOperationsPath, \
+           Configuration.clearingOperationsInputGDB, Configuration.clearingOperationsURL)
+
+        if (self.scratchGDB == None) or (not arcpy.Exists(self.scratchGDB)):
+            self.scratchGDB = UnitTestUtilities.createScratch(Configuration.clearingOperationsPath)
+
+        # set up inputs
+        self.pointTarget = os.path.join(Configuration.clearingOperationsInputGDB, r"CenterPoint")
+        self.inputArea = os.path.join(Configuration.clearingOperationsInputGDB, r"AO")
+
+        UnitTestUtilities.checkFilePaths([Configuration.clearingOperationsPath])
+
+        UnitTestUtilities.checkGeoObjects([Configuration.clearingOperationsInputGDB, self.toolboxUnderTest, self.scratchGDB, self.pointTarget, self.inputArea])
+
+    def tearDown(self):
+        if Configuration.DEBUG == True: print("         ClearingOperationsPointTargetTestCase.tearDown")
+        UnitTestUtilities.deleteScratch(self.scratchGDB)
+
+    def testClearingOperationsPointTarget(self):
+        if Configuration.DEBUG == True:print(".....ClearingOperationsPointTargetTestCase.testClearingOperationsPointTarget")
+        print("Importing toolbox...")
+        arcpy.ImportToolbox(self.toolboxUnderTest)
+        arcpy.env.overwriteOutput = True
+
+        #inputs
+        numCellsH = 9
+        numCellsV = 9
+        cellWidth = 100
+        cellHeight = 100
+        cellUnits = "Meters"
+        labelStart = "Lower-Left"
+        labelStyle = "Alpha-Numeric"
+        output = os.path.join(self.scratchGDB, "ptTarget")
+
+        #Testing
+        runToolMsg="Running tool (Point Target)"
+        arcpy.AddMessage(runToolMsg)
+        Configuration.Logger.info(runToolMsg)
+
+        #try:
+
+        arcpy.PointTargetGRG_ClearingOperations(self.pointTarget, numCellsH, numCellsV, cellWidth, cellHeight, "Meters", None, labelStart, labelStyle, output)
+        #except arcpy.ExecuteError:
+            #UnitTestUtilities.handleArcPyError()
+        #except:
+            #UnitTestUtilities.handleGeneralError()
+
+        result = arcpy.GetCount_management(output)
+        count = int(result.getOutput(0))
+        print("number features: " + str(count))
+        #self.assertEqual(count, 81)
```

Comment:
Unfortunately in @pHill5136 's _arcmap.tbx version the tool isn't showing up in 10.3.1 so it may have been edited on a later version - so still haven't gotten a completely working version of everything on 10.3.1

![image](https://user-images.githubusercontent.com/3090809/29124176-2afc5f0a-7ce6-11e7-9de0-1f6309ec9b72.png)